app/controllers/register.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  willDestroy() {
+    var user = this.get('model');
+    this.store.unloadRecord(user);
+  },
+
+  actions: {
+    createUser() {
+      var user = this.get('model');
+      user.save()
+        .then(() => {
+          this.transitionToRoute('login');
```

Comment:
Also show a notification (using `notify`) that says 
`Welcome <user-name>. Please login to continue.`